docs/src/tut/nft/index.md

```diff
@@ -0,0 +1,739 @@
+1. A step by step guide and explanation of components used while [building](#building-a-dapp) a Reach application.
+
+# Reach Architecture
+
+Using a high-level language such as Reach to build DApps instead of low-level assembly language is attractive for a number of reasons. 
+First, abstraction is a developer's friend. 
+Abstraction allows developers to focus on what really matters in their application, 
+such as, codifying the business logic of their program, 
+rather than determining where the application should access memory.
+
+Another reason is that most humans, even most engineers, 
+are not good at writing low-level code that is efficient, secure, and free of flaws. 
+Reach abstracts contract storage, protocol diagrams, state validation and network details so you can focus on how participants, users of the DApp, interact with one another and the contract.
+
+Reach's backend is very often one single Reach `rsh` file, which is syntactically the same as a JavaScript file, 
+but contains web3 functionality.
+The frontend in a Reach DApp can be created with JavaScript using Reach's [Standard JavaScript Library](https://docs.reach.sh/frontend/#ref-frontends).
+Alternatively, developers can use Python, Go, or C# in their frontend by utilizing Reach's [RPC Server](https://docs.reach.sh/rpc/).
+
+Reach's backend, often named `index.rsh`, 
+instructs data to be published to the consensus network (a blockchain) and defines interfaces that can interact with the frontend. 
+This interface is commonly referred to as a Participant Interact Interface. 
+The frontend defines the interact objects and is often contained within a single file called `index.mjs`. 
+The frontend provides a graphical representation of the participants, e.g. Alice and Bob. 
+The user interface may be in the form of a command line, 
+or more preferably, a web or mobile app.
+
+In Reach, developers think about their participants and how they will interact with one another and the contract. 
+Developers codify the business logic of their application and the Reach compiler assembles the bytecode and smart contract. 
+Reach abstracts the heavy lifting of smart contract development, 
+allowing developers to focus their energy on participant interaction. 
+
+Additionally, Reach's built in theorem verification engine ensures that the contract has the ability to exit, 
+protects against common blockchain attacks, 
+guarantees the contract exits with a zero balance, 
+thus preventing locked away and inaccessible tokens, 
+verifies the contract won't overspend, 
+confirms that formulas are true whether participants and frontends are honest or dishonest, 
+and facilitates token linearity. 
+
+As a result, auditing Reach DApps is easier and less expensive when compared to low-level alternative programming languages. 
+Reach's compiler is more efficient than most human-built low-level code. 
+Altogether, Reach is the smartest, fastest, and safest way to build web3 decentralized applications.
```

Comment:
I bet they'd not want this sentence in it :)

Reply:
You should see what is in there, currently...

"With Reach, the automatic verification of a dApp, alone, is one great reason to consider using Reach vs PyTeal."

I toned the language down!